Library/Homebrew/cask/artifact/abstract_uninstall.rb

```diff
@@ -326,7 +326,12 @@ def each_resolved_path(action, paths)
             next
           end
 
-          yield path, Pathname.glob(resolved_path)
+          begin
+            yield path, Pathname.glob(resolved_path)
+          rescue
```

Comment:
This is too generic. We need to check if full disk access is actually disabled/enabled instead of failing with the same message on any error.

Reply:
Do you just mean add an error type after the `rescue` so it only catches this specific issue as supposed to all exceptions? If so, I agree. I will need to figure out what error is thrown (which shouldn't be too difficult) but I can make that change.

Reply:
```suggestion
          rescue Errno::EPERM 
```

@reitermarkus is this what you mean?